Failed to build with feature "nightly"

```
cargo build --features nightly
   Compiling lru v0.7.6 (/Users/tianyizhuang/Projects/lru-rs)
error[E0277]: the trait bound `K: NotKeyRef` is not satisfied
    --> src/lib.rs:481:38
     |
481  |         if let Some(node) = self.map.get_mut(&k) {
     |                                      ^^^^^^^ the trait `NotKeyRef` is not implemented for `K`
     |
note: required because of the requirements on the impl of `Borrow<K>` for `KeyRef<K>`
    --> src/lib.rs:116:12
     |
116  | impl<K, D> Borrow<D> for KeyRef<K>
     |            ^^^^^^^^^     ^^^^^^^^^
note: required by a bound in `HashMap::<K, V, S, A>::get_mut`
    --> /Users/tianyizhuang/.cargo/registry/src/github.com-1ecc6299db9ec823/hashbrown-0.11.2/src/map.rs:1080:12
     |
1080 |         K: Borrow<Q>,
     |            ^^^^^^^^^ required by this bound in `HashMap::<K, V, S, A>::get_mut`
help: consider further restricting this bound
     |
224  | impl<K: Hash + Eq + NotKeyRef, V, S: BuildHasher> LruCache<K, V, S> {
     |                   +++++++++++

For more information about this error, try `rustc --explain E0277`.
error: could not compile `lru` due to previous error
```

Is the feature still working? Should we remove it? I'd like to submit a PR.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -478,7 +478,7 @@ impl<K: Hash + Eq, V, S: BuildHasher> LruCache<K, V, S> {
     where
         F: Fn() -> V,
     {
-        if let Some(node) = self.map.get_mut(&k) {
+        if let Some(node) = self.map.get_mut(&KeyRef { k: &k }) {
             let node_ptr: *mut LruEntry<K, V> = &mut **node;
 
             self.detach(node_ptr);

